algorithms/C/tree/HeightOfTree.c

```diff
@@ -0,0 +1,77 @@
+// Wap to find the height in a binary search tree.

```

Comment:
What the meaning of  `wap` here i think there should be `way`

Reply:
I think it is abbreviation for **W**rite **A** **P**rogram ( W.A.P). If that's the case then it needs to be capitalised properly.

Reply:
ok @Arsenic-ATG

Reply:
wap is abbreviation for write a program